client/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "drools",
   "displayName": "drools",
   "description": "",
+  "publisher": "kie-group",
```

Comment:
The `kie-group` publisher is associated with a token that we own. When publishing this extension to VS Code marketplace (or even Open-VSX), you will need to use the same token. Another option for you guys to consider is to use the `redhat` publisher. Either way, there is a particular infra that does this publishing task.

Reply:
Thank you, @caponetto 

This project is an upstream effort, so I think `kie-group` is suitable. I will communicate with the tools team and follow the same publication process.